sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -66,6 +68,11 @@ internal object KlaviyoApiClient : ApiClient {
         enqueueRequest(PushTokenApiRequest(token, profile))
     }
 
+    override fun enqueueAggregateEvent(payload: AggregateEventPayload) {
+        Registry.log.verbose("Enqueuing Aggregate Event request")
+        enqueueRequest(AggregateEventApiRequest(payload))
+    }
+
     override fun enqueueUnregisterPushToken(apiKey: String, token: String, profile: Profile) {
         enqueueRequest(UnregisterPushTokenApiRequest(apiKey, token, profile))
```

Comment:
unrelated: mind adding this while you're here? 
```suggestion
        Registry.log.verbose("Enqueuing unregister token request")
        enqueueRequest(UnregisterPushTokenApiRequest(apiKey, token, profile))
```

Reply:
makes sense to me